UKHO.PeriodicOutputService/UKHO.BESS.ConfigurationService/Services/ConfigurationService.cs

```diff
@@ -1,8 +1,10 @@
 ﻿using System.Diagnostics.CodeAnalysis;
+using FluentValidation.Results;
 using Microsoft.Extensions.Logging;
 using NCrontab;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
+using UKHO.BESS.ConfigurationService.Validation;
```

Comment:
Can we improve Stryker score of this class ?

Reply:
Resolved. Thanks